client/ayon_houdini/startup/otls/ayon_lop_import.hda/ayon_8_8Lop_1lop__import_8_81.0/Help

```diff
@@ -0,0 +1,16 @@
+= AYON Load Asset =
+
+#icon: path/to/icon
```

Comment:
```suggestion
#icon: opdef:/ayon::Lop/lop_import::1.0?AYON_icon.png
```
This works on my side
![image](https://github.com/user-attachments/assets/1eb646ed-bc81-4ee4-981b-cb439b61f998)

Reply:
It's so weird - because this just doesn't want to work for me somehow. I'm on Hou 20 though.

Anyway, may just be a 'refresh' error or whatever because I also often see the unchanged text instead of what I expected to be there after changing things.